### PR TITLE
Safaricom Mpesa Daraja (Independent Publisher)

### PR DESCRIPTION
**Pull request by muchirimaina**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by muchirimaina** [comment]
Test Operations Are Running Successfully.
daraja1

The Paconn Validation was successful.
daraja2

STK push runs successfully.
daraja3

A flow to initiate a payment transaction runs successfully.
daraja4

**Comment by muchirimaina** [comment]
Hi @vmanoharas 👋🏽 
I have created a new pull request (PR). Please review and advise accordingly. Additionally, I request that you add bkaraba14@gmail.com as my co-collaborator on this connector.
Thank you!😊

File: independent-publisher-connectors/Safaricom Mpesa Daraja/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Daraja api connector
```
Comment (vmanoharas) [inline independent-publisher-connectors/Safaricom Mpesa Daraja/README.md:1]: Hi @johnmuchiriM, 
Please update readme.md file as per the template in this path. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md

**Review by vmanoharas — COMMENTED** [review]
Hello @johnmuchiriM, 
Hello Partner,
Thank you very much and congratulations on your new submission. 
Can you please resolve review comments?
Thank you very much again for working with us.

**Review by vmanoharas — COMMENTED** [review]
Hi @johnmuchiriM, 

Please modify readme.md files as per the template, headings are not matching with the template.

**Comment by muchirimaina** [comment]
Hi @vmanoharas,
Noted, I will take action and effect the changes requested.

**Comment by vmanoharas** [comment]
> Hi @vmanoharas, Noted, I will take action and effect the changes requested.

Thank you @johnmuchiriM.

**Comment by muchirimaina** [comment]
Hi @vmanoharas,

The APIs that we are tapping into to create the Connector have undergone some recent changes which will change the Authentication structure from the earlier setup we had done. So if you don't mind, kindly give us some time to restructure the setup, and then we will get back to you as soon as we're done.

**Comment by vmanoharas** [comment]
> Hi @vmanoharas,
> 
> The APIs that we are tapping into to create the Connector have undergone some recent changes which will change the Authentication structure from the earlier setup we had done. So if you don't mind, kindly give us some time to restructure the setup, and then we will get back to you as soon as we're done.

Hi @johnmuchiriM, Thank you!

**Comment by vmanoharas** [comment]
Hello @johnmuchiriM, 

I am following up on the update.

**Comment by muchirimaina** [comment]
Hello @vmanoharas, I am still working on it. I had a couple of projects on my desk from my day-to-day work, and I am clearing them up so that I can give full attention to the new changes made on the APIs. I will contact you as soon as the set-up is complete.

**Comment by vmanoharas** [comment]
Hello @johnmuchiriM, 

I am following up on the update, as this PR is open from more than 5 months.

**Comment by muchirimaina** [comment]
Hi @vmanoharas! I've been swamped with internal projects lately. Fortunately, with some major releases this month, I anticipate having time for our connector going forward. Thanks for your patience!

**Comment by vmanoharas** [comment]
Hello @johnmuchiriM, 

This PR is in open state from many months, I am closing this PR, kindly re-open/create new PR.

**Pull request closed without merging**